kudu/pom.xml

```diff
@@ -0,0 +1,167 @@
+    <antlr4-runtime-version>4.7</antlr4-runtime-version>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.antlr</groupId>
+        <artifactId>antlr4-maven-plugin</artifactId>
+        <version>4.7</version>
+        <executions>
+          <execution>
+            <id>kudusqlantlr4parsergen</id>
+            <goals>
+              <goal>antlr4</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <configuration>
+          <logViolationsToConsole>${checkstyle.console}</logViolationsToConsole>
+          <excludes>**/KuduSQLExpression*</excludes> <!--To account for violations from the Antlr autogenerated code -->
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.rat</groupId>
+        <artifactId>apache-rat-plugin</artifactId>
+        <configuration>
+          <excludes combine.children="append">
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpression.tokens</exclude>
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpressionLexer.tokens</exclude>
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpressionLexer.java</exclude>
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpressionListener.java</exclude>
+            <exclude>src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpressionParser.java</exclude>
+            <exclude>src/main/antlr4/KuduSQLExpression.g4</exclude>
+          </excludes>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>add-source</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>add-source</goal>
+            </goals>
+            <configuration>
+              <sources>
+                <source>${basedir}/target/generated-sources/antlr4</source>
+              </sources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>malhar-library</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>malhar-library</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+      <classifier>tests</classifier>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>apex-common</artifactId>
+      <version>${apex.core.version}</version>
+      <type>jar</type>
```

Comment:
@ananthc this is the dependency that needs to be scope provided

Reply:
Done.